### ISSUE TITLE
Variables / Galaxy overlap: existing framework for periodic variable recovery

### ISSUE THREAD

**Issue by willclarkson** [LSSTScienceCollaborations/ObservingStrategy#203]
@lmwalkowicz and @AshishMahabal - We (@akvivas, @willclarkson and the Milky Way folks) would like to know a bit more about the framework you currently have for recovery of periodic variables, since a number of our metrics will likely depend on them. (We expect that many of our metrics would call your metrics, see e.g. the current version of Section 4.3.2 for examples.)

When you get time, could you point us to any examples of implementations of metrics you currently have for recovery of periodic variables please?

**Comment by lmwalkowicz** [comment]
hey @willclarkson, @lundmb is currently working on our mess-of-a-section about existing periodic-stuff-related metrics. Mike any updates here? As I understand them currently, most of the extant metrics quantify how survey deployment affects one's ability to recover a period-- e.g. what aliases are introduced into the periodogram of a given object by the observing strategy. So in other words, they are largely agnostic of specific period and may therefore be interesting to you. A better description of them should exist in the Variables introduction section by later today.

**Comment by lundmb** [comment]
I'm in the process of adding more detailed descriptions in the variables section for the metrics that we have, as well as including the MAF results for each of these metrics using the enigma_1189 cadence.

The quick rundown of the metrics in question:
Periodogram Purity Function: Established in detail in a paper of mine here: http://arxiv.org/pdf/1508.03175v2.pdf
The Periodogram Purity Function is based on the window function of the data and presents a measure for how much signal strength is lost to other frequencies

Phase Gap Metric: In the varMetric contributed metric, this creates a series of random periods, and checks the maximum phase gap in the observing schedule for each period. The visualizations are the median and maximum values for the maximum phase gap for each period. The defaults are 5 periods between 3 and 35 days.

Period Deviation Metric: Also in varMetric, this uses the same defaults as above, and calculates the error in recovering the period of a simple sine function.

**Comment by willclarkson** [comment]
@lmwalkowicz and @lundmb - thanks, this helps!! @lundmb , am I correct in thinking that the metrics you mention are implemented in the LSST-nonproject/sims_maf_contrib/mafContrib subdirectory on github?

**Comment by lundmb** [comment]
Yes, they're located there; I'm going to be including figures showing the output for them, as well (The only one I'm still running is the Period Deviation Metric)

**Comment by willclarkson** [comment]
I think this discussion has been overtaken by sims_maf_contrib development and also by Chapters 5 & 6... I'm punting this for the Tucson milestone in case there are undocumented features coming soon!

**Comment by AshishMahabal** [comment]
Agreed. Unless @lundmb has additional comments, this can be closed ( @lmwalkowicz )
